src/Services/HealthCheckService.php

```
@@ -200,19 +213,34 @@ private function formatBytes(int $bytes, int $precision = 0): string
+     * @noinspection SqlNoDataSourceInspection
+     */
     private function checkMigrations(): array
     {
-        $pendingMigrations = DB::select('SELECT * FROM migrations');
-
-        return [
-            'total_migrations' => count($pendingMigrations),
-        ];
+        $table = config('database.migrations.table');
+        try {
+            $pendingMigrations = DB::select("SELECT * FROM $table");
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\DB' in method 'checkMigrations'. (undefined)

(StaticAccess)